scripts/deploy/deploy-dashboard.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Go to the project root directory
+cd $(dirname ${0})/../..
+
+# Paths to the dashboard and functions directories.
+dashboardFolder=tools/dashboard
+
+# Go to the dashboard folder because otherwise the Firebase CLI tries to deploy the wrong project.
+cd ${dashboardFolder}
+
+# Install node_modules for the dashboard and afterwards build the dashboard app in production.
+(npm install; npm run build-production) &
```

Comment:
Can we make this just `ng build --aot --prod` here? Everyone knows that that does but the `npm run` adds a level of indirection